content/en/docs/setup/install/helm/index.md

```diff
@@ -0,0 +1,160 @@
+    {{< /text >}}
+
+1. Install the Istio base chart which contains cluster level resources like
+   Custom Resource Definitions (CRDs), cluster role and cluster role bindings
+   and service accounts used by the Istio components:
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istio-base manifests/charts/base
+    {{< /text >}}
+
+1. Install the Istio discovery (istiod) chart which includes the Istio control
+   plane deployment and its associated configuration:
+
+    {{< warning >}}
+    The default chart configuration uses the secure third party tokens for Service
+    Account token projections used by Istio proxies to authenticate with the Istio
+    control plane. Before proceeding to install this chart, you should verify
+    if third party tokens are enabled in your cluster by following the steps
+    describe [here](/docs/ops/best-practices/security/#configure-third-party-service-account-tokens).
+    If third party tokens are not enabled, you should add the option
+    `--set global.jwtPolicy=first-party-jwt` to the following command.
+    If the `jwtPolicy` is not set correctly, the `istiod` pod will not get
+    deployed due to the missing `istio-token` volume.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istiod manifests/charts/istio-control/istio-discovery
+    {{< /text >}}
+
+1. (Optional) Install the Istio ingress gateway chart which contains the ingress
+   gateway components:
+
+    {{< warning >}}
+    Similar to the discovery chart above, the default ingress gateway chart uses
+    third party tokens. If third party tokens are not enabled, you should
+    add the option `--set global.jwtPolicy=first-party-jwt` to the following command.
+    If the `jwtPolicy` is not set correctly, the `istio-ingressgateway` pod will
+    not get deployed due to the missing `istio-token` volume.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istio-ingress manifests/charts/gateways/istio-ingress
+    {{< /text >}}
+
+1. (Optional) Install the Istio egress gateway chart which contains the egress
+   gateway components:
+
+    {{< warning >}}
+    Similar to the discovery chart above, the default egress gateway chart uses
+    third party tokens. If third party tokens are not enabled, you should
+    add the option `--set global.jwtPolicy=first-party-jwt` to the following command.
+    If the `jwtPolicy` is not set correctly, the `istio-egressgateway` pod will
+    not get deployed due to the missing `istio-token` volume.
+    {{< /warning >}}
+
+    {{< text bash >}}
+    $ helm install --namespace istio-system istio-egress manifests/charts/gateways/istio-egress
+    {{< /text >}}
+
+## Verifying the installation
+
+1. Ensure all Kubernetes pods in `istio-system` namespace are deployed and have a
+   `STATUS` of `Running`:
+
+    {{< text bash >}}
+    $ kubectl get pods -n istio-system
+    {{< /text >}}
```

Comment:
add istioctl verify-install step?

Reply:
istioctl verify doesn't seem to work with Helm based installs.

```bash
./bin/istioctl verify-install
Error: could not load IstioOperator from cluster: control plane revision "" not found.  Use --filename
```